dart/scopes_and_var.dart

```diff
@@ -0,0 +1,15 @@
+void main() {
+  var a = 0, b = 1, c = 2;
+  (() {
+    var a = 100;   // Local declaration, global a is safe!
+    b = 200;       // Unlike javascript, dart won't overwrite global variables - this is a runtime error
```

Comment:
Dart doesn't really allow you to screw yourself over like this... not sure if this should be left in or not.